keras/utils/vis_utils.py

```diff
@@ -38,6 +39,7 @@ def model_to_dot(model,
                  expand_nested=False,
                  dpi=96,
                  subgraph=False):
+                 #subgraph=True):
```

Comment:
```suggestion
                 #subgraph=True):
```
please remove unnessary comments